extensions/objects/templates/sanitized-url-editor.component.ts

```diff
@@ -16,10 +16,12 @@
  * @fileoverview Directive for sanitized URL editor.
  */
 
-import { ChangeDetectorRef, Component, EventEmitter, Input, Output } from '@angular/core';
+import { AfterViewInit, ChangeDetectorRef, Component, EventEmitter, Input, Output, ViewChild } from '@angular/core';
+import { NgForm } from '@angular/forms';
 import { downgradeComponent } from '@angular/upgrade/static';
 import { SchemaDefaultValue } from 'services/schema-default-value.service';
 
+const VALIDATION_STATUS_INVALID = 'INVALID';
```

Comment:
I'm a bit surprised this constant is here. It sounds quite general -- we don't use it more globally?

Reply:
No we don't. I would actually think we should avoid using it more globally. This is used by angular forms but not provided in the public api. Can't change it now because we need it for validations. That said it has been in the angular forms since 2016, so not really scared about this changing suddenly. This should go away in our restructuring of schema-based-editors. Adding a comment about this in the code might distract new validations from using this. (Also, the only other usage is in schema-based-editor component). I have moved it around to a utility file.